src/components/font_size_editor/font_size_editor.xml

```diff
@@ -0,0 +1,35 @@
+<templates>
+  <t t-name="o-spreadsheet-FontSizeEditor" owl="1">
+    <div class="o-dropdown o-font-size-editor" t-ref="FontSizeEditor">
+      <input
+        type="number"
+        min="1"
+        max="400"
+        class="o-font-size bg-transparent border-0"
+        t-on-keydown="onInputKeydown"
+        t-on-click.stop=""
```

Comment:
We try to avoid that except if it's really needed. Why is it needed ?

Reply:
Because we don't want to close the list when we simply click on the input

Reply:
the input is a child element of `rootEditorRef`, right ?

Reply:
My mistake, this wasn't there to avoid closing the list but to make sure the text is selected with the onfocus